packages/web3torrent/src/utils/magnet.ts

```diff
@@ -1,39 +1,34 @@
 import {defaultTrackers, EmptyTorrent} from '../constants';
 import {RoutePath} from '../routes';
 import {Torrent} from '../types';
+import {useLocation} from 'react-router-dom';
 
-export const parseMagnetURL: (rawMagnetURL: string) => Torrent = (rawMagnetURL = '') => {
-  const emptyTorrentData = {...EmptyTorrent, magnetURI: ''} as Torrent;
-  if (!rawMagnetURL.trim()) {
-    return emptyTorrentData;
-  }
-  const magnetParams = new URLSearchParams(rawMagnetURL.trim().replace(/#magnet:/g, ''));
-  if (!magnetParams.has('xt')) {
-    return emptyTorrentData;
-  }
+export const useQuery = () => {
```

Comment:
I found this a bit confusing at first. Do you think it would be useful to document that this is the recommended approach in the official docs: https://reacttraining.com/react-router/web/example/query-parameters